lib/components/transition-fade-in-out/index.tsx

```diff
@@ -1,9 +1,17 @@
-import React from 'react';
+import React, { FunctionComponent } from 'react';
 import PropTypes from 'prop-types';
 import { CSSTransition } from 'react-transition-group';
 
-const TransitionFadeInOut = props => {
-  const { children, wrapperClassName, shouldMount } = props;
+type OwnProps = {
```

Comment:
`children` should be `ReactElement` or `Element` or `ReactNode` or something like that - I think `ReactNode`